gitrevise/todo.py

```diff
@@ -244,11 +244,23 @@
 
 def apply_todos(
     current: Optional[Commit],
+    original: List[Step],
     todos: List[Step],
```

Comment:
Instead of `(original, todos)` I'd call them `(todos, reordered_todos)`

Reply:
How about `(todos_original, todos_edited)`? Ideally, the names should be the same in the caller:

```py
# Build up an initial todos list, edit that todos list.
todos = original = build_todos(to_rebase, staged)

if enable_autosquash(args, repo):
    todos = autosquash_todos(todos)

if args.interactive:
    todos = edit_todos(repo, todos, msgedit=args.edit)

if todos != original:
    # Perform the todo list actions.
    new_head = apply_todos(base, original, todos, reauthor=args.reauthor)
```

Reply:
yeah that would work well. `todos` is used more often so it can get the shorter name.
My main point is that both should have "todos" in their name